WordPress/src/main/java/org/wordpress/android/ui/people/PeopleInviteFragment.java

```diff
@@ -194,17 +196,26 @@ public void onFocusChange(View v, boolean hasFocus) {
             populateUsernameButtons(usernames);
         }
 
-        View roleContainer = view.findViewById(R.id.role_container);
+        View roleContainer = view.findViewById(R.id.role);
```

Comment:
It seems that the local variable (`View roleContainer`) is no longer needed and we could reconcile the duplicate `findViewById(R.id.role)` with the mRoleTextView one. WDYT?